doc/src/documentation/architecture/security/README.md

```diff
@@ -2,29 +2,35 @@
+## Features and the Personas that can use it
 
 - [Nachwuchskräfte](../../../features/Nachwuchskraefte.md) (örtliche Ausbildungsleitung)
 - [Meldezeitraum](../../../features/meldezeitraum.md) (örtliche Ausbildungsleitung)
-- [Meldung örtl. Ausbilder](../../../features/Praktikumsplaetze.md) (örtliche Ausbilderin / anyone except Nwk)
-- [Meldung örtl. Ausbildungsleitung](../../../features/Praktikumsplaetze.md) (örtliche Ausbildungsleitung)
-- [Zuweisung](../../../features/Zuweisung.md) (örtliche Ausbildungsleitung)
+- [Submission örtl. Ausbilderin](../../../features/Praktikumsplaetze.md#submission-örtliche-ausbilderin) (örtliche Ausbilderin)
+- [Submission örtl. Ausbildungsleitung](../../../features/Praktikumsplaetze.md#submission-örtliche-ausbildungsleitung) (örtliche Ausbildungsleitung)
+- [Assignment](../../../features/Zuweisung.md) (örtliche Ausbildungsleitung)
 
 ## Technical Implementation
 On the frontend side, the security features are controlled by the `VITE_APP_SECURITY` in the `.env` files.
 By default, security features are enabled in `production` and disabled in `development`.
 
-On the frontend side, we have a `v-security` directive. This directive has two modifiers: `allow` and `restrict`.
-`allow` is a whitelisting mechanism, so the role that is used as a parameter with `allow` can see the content displayed underneath the tag.
-`restrict` is a blacklisting mechanism, so the role that is used as a parameter with `restrct` cannot see the content displayed underneath the tag.
-The reason for black- and whitelisting is that there is the persona [Ausbilder](../../../glossary.md#ortliche-ausbilderin) which is applicable to every person who is not a [Nwk](../../../glossary.md#nwk).
-It would be unreasonable to give this role to every employee that is not a [Nwk](../../../glossary.md#nwk), so we decided not to create a role for this, but instead
-use this black- and white-listing, depending on the other roles.
\ No newline at end of file
+On the frontend side, we have a `security composable`.
+This composable has several methods:
+ - checkForRole(string): this method checks whether the logged-in user has the provided role
+ - checkForAnyRole(string[]): this method checks whether the logged-in user has any of the provided roles
+ - checkForAllRoles(string[]): this method checks whether the logged-in user has all provided  roles
+ - isAusbildungsleitung(): This method is used for checking whether the logged-in user is a [Ausbildungsleitung](../../../glossary.md#ortliche-ausbildungsleitung)
+
+Those methods are used in `v-if` directives to control what is shown (rendered) to the user.
+As this is controlled on frontend side in the browser, users could possibly bypass this security
+so this is not to be meant to prevent users from doing things that are not allowed, but more a way
+to guide them in the application and only show them things they is allowed to do and not confuse
+them with many error messages due to missing privileges.
```

Comment:
```suggestion
As this is controlled on frontend side in the browser, users could possibly bypass this security
so this is not meant to prevent users from doing things that are not allowed, but more a way
to guide them in the application and only show them things they are allowed to do and not confuse
them with many error messages due to missing privileges.
```